src/Microsoft.SqlTools.ServiceLayer/Admin/AdminService.cs

```diff
@@ -163,7 +163,7 @@ await requestContext.SendResult(new GetDatabaseInfoResponse()
                     });
                 };
 
-                Task.Run(() => requestHandler());
+                Task task = Task.Run(async () => await requestHandler());
```

Comment:
You need to add the `.ContinueOnFaulted` extension method to any task kicked off by `Task.Run`, or it will crash the process on an unexpected failure. You should add the code to do a `SendError` as part of this method - currently your try/catch doesn't really do anything since it won't actually catch errors inside the `requestHandler = async ...` function.